exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporterConfiguration.java

```diff
@@ -168,16 +174,27 @@ public String toString() {
   }
 
   public static class AuthenticationConfiguration {
-    public String username;
-    public String password;
+    private String username;
+    private String password;
 
     public boolean isPresent() {
       return (username != null && !username.isEmpty()) && (password != null && !password.isEmpty());
     }
 
-    @Override
-    public String toString() {
```

Comment:
I would propose to still override `toString` and keep it blank or something, with a comment that we should not implement it. That way it's less likely that we later add it accidentally. WDYT?